Resources/Private/Language/de.locallang.xlf

```diff
@@ -591,6 +591,10 @@ Sächsische Landesbibliothek- Staats- und Universitätsbibliothek Dresden
 			<source>Type</source>
 			<target>Dokumenttyp</target>
 		</trans-unit>
+		<trans-unit id="search.form.placeholder">
+			<source>yyyy-mm-dd</source>
+			<target>tt.mm.jjjj</target>
```

Comment:
I prefer upper case in this place. But this may be technician view of life.

Reply:
Should not be a problem...